tooling/scripts/docs-upload.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright 2023 Impulse Innovations Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import os
+import shutil
+from pathlib import Path
+from zipfile import ZipFile
+
+import requests
+from docs_builder.reference_docs_build import build
+from requests.auth import HTTPBasicAuth
+
+os.mkdir('__docs')
+
+package = os.environ['PACKAGE']
+version = os.environ['VERSION']
+
+# Build reference docs
+if os.environ.get('BUILD_REFERENCE') == 'true':
+    package_paths = os.environ.get('PACKAGE_PATHS', '').strip().split('\n')
```

Comment:
@sam-causalens I could not test it without the secrets but I figured this was still OK even though we only have a single path (unlike the multiple paths for the `dara` monorepo). Just flagging as I was unsure here. But given we will have no new-line then the final split just won't happen.